src/App.js

```diff
@@ -1,6 +1,43 @@
 import React from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination/Pagination';
+import { PerPageSelector } from './components/PerPageSelector/PerPageSelector';
 
-const App = () => <h1>Pagination</h1>;
+export class App extends React.Component {
+  state = {
+    page: 1,
+    perPage: 5,
+    perPageOptions: [3, 5, 10, 20],
+    withInfo: true,
```

Comment:
what is the purpose of this variable

Reply:
There is a requirement "Add an optional param withInfo to show extra info before the buttons (6 - 10 of 20)". I had no clear idea of what this should look like. I agree that 'withInfo' shouldn't be part of state as there is no way to change it in the app. So I've excluded it from the state and just pass it to the Pagination component as a boolean attribute. If there is no need for additional information one can omit to pass this attribute to the Pagination component.